bat/tests/build-multiple-delta-packs/run.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+
+# shared test functions
+load ../../lib/mixerlib
+
+setup() {
+  global_setup
+}
+
+@test "build delta-packs after build all for different versions" {
+  current_ver=$(get-current-version)
+  mixer-init-stripped-down "$current_ver" 10
+  sudo -E mixer build all --native
+  sudo mixer versions update
+  mixer-build-all --native
+  sudo -E mixer build delta-packs --native --previous-versions=10
+  sudo rm -rf ./update/image/10/full/usr/lib/os-release
+  sudo mixer versions update
+  sudo -E mixer build all --native
+  sudo -E mixer build delta-packs --native --previous-versions=10
+  test $(< mixversion) -eq 30
+
+  var1=`[ "$(ls -A ./update/www/10/delta)" ] && echo "Not Empty" || echo "Empty"`
+  test "$var1" = "Empty"
```

Comment:
Not necessarily wrong, but since ls will already give you a binary value that will be checked by test, you can you that value directly instead of assigning strings to it. You can use something like `test $(ls -A1 | wc -l) -eq 0` for example.
It is fine as is though,  no need to change if you believe this is easier to read.